common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/StatusReport.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018 LINE Corporation. All rights reserved.
+ * LINE Corporation PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ */
+
+package com.linecorp.line.auth.fido.fido2.common.mdsv3;
+
+import lombok.Data;
+
+@Data
+public class StatusReport {
```

Comment:
This structure is missing the `authenticatorVersion`: https://fidoalliance.org/specs/mds/fido-metadata-service-v3.0-ps-20210518.html#dictdef-statusreport

Reply:
Fixed in 17a944d.